### DISCUSSION TITLE
Power Supply

### DISCUSSION THREAD

**Discussion by beau-seidon** [Jaffe-/NESizer2#20] [Q&A] [answered]
The front panel schematic recommends a minimum of 8 V, but the 7805 can (allegedly) handle all the way up to 35 V input. 

I have been using a 12 V power supply but my heat sink gets pretty warm dissipating the excess. I'm considering using a 9 V supply. 

Has anyone encountered issues using a voltage below 12 V? 

Has anyone tried powering it with a 9 V battery and a barrel jack adapter, for portability?

**Comment by Jaffe-** [comment, marked as answer]
I've been running it on 9 V, that works just fine! So a 9 V battery should also work, but I guess the question is for how long. I haven't measured the average current consumption of the NESizer, but I suspect it is not exactly power efficient.

**Comment by jeff464** [comment]
For me, a brand new Panasonic 9v battery lasted about ten minutes before the unit started crashing. 

For power supplies, is it center positive or negative? I assume this is DC? What's the amperage?
